.github/ISSUE_TEMPLATE.md

```diff
@@ -1,9 +1,13 @@
 <!--
-Thank you for reporting an issue! Please review the following notes before submitting it.
+Thank you for reporting an issue!
+Please review the following notes before submitting it.
 
-Most issues on this repo are requests for new commands. If it's the case, make sure to use the standard title ("page request: <command name>"), and include a link to a web page about the command, and if possible, to an existing source of example-style documentation.
+Most issues on this repo are requests for new commands.
+If the is the case, make sure to use the standard title ("page request: <command name>") and include a link to a web page about the command and, if possible, to an existing source of example-style documentation.
 
-If your issue is with a particular client of tldr-pages, please raise the issue in the repo for that client. For example, if you are using the node client, you would report the issue here: https://github.com/tldr-pages/tldr-node-client/issues
+If your issue is with a particular tldr client, please report the issue there.
+For example, issue for the node client should be reported at https://github.com/tldr-pages/tldr-node-client/issues
```

Comment:
This wording change isn't helpful I don't think.

Reply:
You cannot raise an issue "in the repo". It might have a separate bug tracker or whatever.

Reply:
```suggestion
For example, issue for the Node.js client should be reported at https://github.com/tldr-pages/tldr-node-client/issues
```